pkg/api/v1/types.go

```diff
@@ -1023,6 +1023,54 @@ type ReplicationControllerList struct {
 	Items []ReplicationController `json:"items" description:"list of replication controllers; see http://releases.k8s.io/HEAD/docs/replication-controller.md"`
 }
 
+// DaemonSpec is the specification of a daemon.
+type DaemonSpec struct {
```

Comment:
Add the kinds to ignoredKinds in pkg/api/latest/latest.go. That should prevent them from being exposed.